drivers/src/mailbox.rs

```diff
@@ -360,6 +376,27 @@ impl MailboxRecvTxn<'_> {
         Ok(())
     }
 
+    /// Writes buf.len() bytes to the mailbox datain reg as dwords
+    fn enqueue(&mut self, buf: &[u8]) -> CaliptraResult<()> {
+        MailboxTxFifo::try_enqueue(self.mbox, buf)
+    }
+
+    /// Writes number of bytes to data length register.
+    pub fn write_dlen(&mut self, dlen: u32) -> CaliptraResult<()> {
```

Comment:
Why is this public? We don't anyone from the outside calling it.

Reply:
Thanks! This was an error on my part.